controllers/routing/routing_controller.go

```diff
@@ -131,3 +134,7 @@ func (r *PlatformRoutingController) Activate() {
 func (r *PlatformRoutingController) Deactivate() {
 	r.active = false
 }
+
+func IsMarkedForDeletion(target *unstructured.Unstructured) bool {
```

Comment:
Depending on which PR comes first, but we should move this to the new `pkg/unstruct/funcs.go`  which is introduced in  #60

Reply:
done https://github.com/maistra/odh-platform/pull/48/commits/5c0cae9955ba0302d18482c16a40c7cba040af95